src/main/java/seedu/fridgefriend/FridgeFriend.java

```diff
@@ -4,20 +4,24 @@
 import seedu.fridgefriend.exception.InvalidInputException;
 import seedu.fridgefriend.food.Fridge;
 import seedu.fridgefriend.utilities.Parser;
+import seedu.fridgefriend.utilities.Save;
 import seedu.fridgefriend.utilities.Ui;
 
 public class FridgeFriend {
 
     private static boolean isExit = false;
+    private static Fridge fridge = new Fridge();
```

Comment:
All the methods in Fridge are static, there is no need to declare a fridge variable here